### PR TITLE
feat(examples): Add the hello world example for gemm.

### PR DESCRIPTION
**Pull request by haruhi55**
Added a "Hello World" example using GEMM.

As the current implementation of the global-to-register loader still requires enhancements to be compatible with TileIterator, this PR initializes the GEMM example with the simplest case.

### CODE REVIEW

File: examples/gemm/gemm.cu
Comment (KuangjuX) [inline examples/gemm/gemm.cu:1]: Is it necessary to differentiate between different languages (C++ and Python) in the `examples` directory?

**Review by KuangjuX — COMMENTED** [review]
Almost Approved.

File: examples/gemm/gemm.cu
Comment (haruhi55) [reply examples/gemm/gemm.cu:1]: Make sense. How about creating two directories, `python` and `cpp`, to organize the examples?

Comment (haruhi55) [reply examples/gemm/gemm.cu:1]: > Is it necessary to differentiate between different languages (C++ and Python) in the `examples` directory?

Done.

**Review by KuangjuX — APPROVED** [review]
LGTM!

**Pull request merged**